public/app/plugins/datasource/cloudwatch/migrations.ts

```diff
@@ -122,7 +122,7 @@ export function migrateVariableQuery(rawQuery: string | VariableQuery): Variable
     newQuery.namespace = dimensionValuesQuery[2];
     newQuery.metricName = dimensionValuesQuery[3];
     newQuery.dimensionKey = dimensionValuesQuery[4];
-    newQuery.dimensionFilters = dimensionValuesQuery[6] || '';
+    newQuery.valueDimensions = dimensionValuesQuery[6] ? JSON.parse(dimensionValuesQuery[6]) : {};
```

Comment:
Do we need a try catch around the `JSON.parse` or is `dimensionValuesQuery[6]` always a well formed JSON object?